package.json

```diff
@@ -53,7 +53,7 @@
   "dependencies": {
     "@react-spring/three": "9.6.1",
     "@react-three/fiber": "8.13.5",
-    "@use-gesture/react": "^10.3.1",
+    "@use-gesture/react": "10.3.1",
```

Comment:
What happened here ?

Reply:
changed to fixed version
I've tested different version of libs when trying prevent issue with sticky dragging